webpack.config.js

```diff
@@ -17,8 +17,8 @@ const path = require('path');
 module.exports = {
   entry: './src/index.ts',
   output: {
-    library: 'BigQueryStorage',
-    filename: './big-query-storage.js',
+    library: 'BigQueryRead',
```

Comment:
This client rename doesn't feel right.  It's different from the name of the library in a bizarre way.  Any ideas what caused that?

Reply:
Originally they thought it would be a unified "storage" client in the original v1beta1.  What's actually happening is a they're creating two clients, a read and a write client

Reply:
@steffnay is the idea that this repository would have both the `read` and `write` client, if so I think we should probably coordinate with Alex and Xiaozhen one what this structure would look like with the new micro generator.

Reply:
@bcoe yes, that's the plan